CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 0.4.2
+
+- Support JS IR backend
+- Support Gradle 7.0 and newer [#67](https://github.com/Kotlin/kotlinx-benchmark/issues/67)
+- Make `mode` configuration parameter work with values considered valid in README.MD
```

Comment:
Maybe make it more clear that this is a breaking change, so previous usages of this configuration parameter should be updated.

Reply:
ah, ok